libs/moment-service/src/lib/moment-service.tsx

```diff
@@ -18,8 +18,8 @@ export const deadlineToDate = (deadline: string, dateFormat = 'DD MMM') => {
   return moment(deadline, 'x').format(dateFormat);
 };
 
-export const startToDate = (start: moment.Moment, dateFormat = 'DD MMM') => {
-  return moment(start).format(dateFormat);
+export const momentToDate = (m: moment.Moment, dateFormat = 'DD MMM') => {
+  return moment(m).format(dateFormat);
```

Comment:
This needs vopy as it modifies previous value, which we probably don't need?

(Momentjs works inplace)